tests/outputs/a_presentation/expenses.yaml

```diff
@@ -1,54 +1,3 @@
-2006as_mars:
```

Comment:
please can you redo the tests in a way that doesn't require this to be deleted?  This is causing test coverage to go down, not up, which is the desired outcome.

Reply:
> Yes. I am thinking that I can approach this as follows:
> Subtract nothing from the .yaml files inside the outputs/a_presentations/ directory, and leave the original hm item as it was before. Instead, I'll add a new entry to hm dedicated to testing a case where someone uses the `--expense_db` option. Correspondingly, the data would go into presentations.yaml but _not_ expenses.yaml.

On first paragraph, yes, you can try.  Not sure if there is a way to do it where the second call doesn't actually update the presentations yml or it won't work.

For the second paragraph, we kind of painted ourself into a corner and the cost of getting out of it is high.  We would have to create multiple databases because all the adder/updater helpers do the same.  Don't go there or it is all you will do this summer..... 

I think for this case, just reverting the tests to what they were, and not testing this current functionality is the best, even if it makes part of me die....